config/config.go

```diff
@@ -201,6 +203,13 @@ func (cfg *Config) NewNode(ctx context.Context) (host.Host, error) {
 		}
 	}
 
+	if cfg.EnableAutoRelay || cfg.AutoNATOpts != nil {
```

Comment:
What sets AutoNATOpts?

Reply:
IPFS?
Should there be a libp2p translation of the nat options?